dynatrace-oneagent-operator/README.md

```diff
@@ -39,6 +39,27 @@ Generate an API and a PaaS token in your Dynatrace environment.
 
 https://www.dynatrace.com/support/help/reference/dynatrace-concepts/why-do-i-need-an-environment-id/#create-user-generated-access-tokens
 
+#### Apply CustomResourceDefinition
```

Comment:
Would remove one # here and in the subheaders so it's the same like 'Prepare tokens'

Reply:
Adjusted